pkg/config/lima_config_applier_darwin_test.go

```diff
@@ -1,6 +1,9 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
+//go:build darwin
+// +build darwin
```

Comment:
nit: `// +build <tag>` tags obsolete after Go 1.18 https://tip.golang.org/doc/go1.18 . this applies to a few other places in this PR. `go fix` should remove them automatically for you